packages/deploy-configurations/configs/base.conf.ts

```diff
@@ -0,0 +1,1211 @@
+      name: 'RWA003',
+      address: '0x0000000000000000000000000000000000000000'
+    },
+    RWA004: {
+      name: 'RWA004',
+      address: '0x0000000000000000000000000000000000000000'
+    },
+    RWA005: {
+      name: 'RWA005',
+      address: '0x0000000000000000000000000000000000000000'
+    },
+    RWA006: {
+      name: 'RWA006',
+      address: '0x0000000000000000000000000000000000000000'
+    },
+    SDAI: {
+      name: 'SDAI',
+      address: '0x0000000000000000000000000000000000000000'
+    },
+    STETH: {
+      name: 'STETH',
+      address: '0x0000000000000000000000000000000000000000'
+    },
+    TBTC: {
+      name: 'TBTC',
+      address: '0x0000000000000000000000000000000000000000'
+    },
+    TUSD: {
+      name: 'TUSD',
+      address: '0x0000000000000000000000000000000000000000'
+    },
+    UNI: {
+      name: 'UNI',
+      address: '0x0000000000000000000000000000000000000000'
+    },
+    UNIV2AAVEETH: {
+      name: 'UNIV2AAVEETH',
+      address: '0x0000000000000000000000000000000000000000'
+    },
+    UNIV2DAIETH: {
+      name: 'UNIV2DAIETH',
+      address: '0x0000000000000000000000000000000000000000'
+    },
+    UNIV2DAIUSDC: {
+      name: 'UNIV2DAIUSDC',
+      address: '0x0000000000000000000000000000000000000000'
+    },
+    UNIV2DAIUSDT: {
+      name: 'UNIV2DAIUSDT',
+      address: '0x0000000000000000000000000000000000000000'
+    },
+    UNIV2ETHUSDT: {
+      name: 'UNIV2ETHUSDT',
+      address: '0x0000000000000000000000000000000000000000'
+    },
+    UNIV2LINKETH: {
+      name: 'UNIV2LINKETH',
+      address: '0x0000000000000000000000000000000000000000'
+    },
+    UNIV2UNIETH: {
+      name: 'UNIV2UNIETH',
+      address: '0x0000000000000000000000000000000000000000'
+    },
+    UNIV2USDCETH: {
+      name: 'UNIV2USDCETH',
+      address: '0x0000000000000000000000000000000000000000'
+    },
+    UNIV2WBTCDAI: {
+      name: 'UNIV2WBTCDAI',
+      address: '0x0000000000000000000000000000000000000000'
+    },
+    UNIV2WBTCETH: {
+      name: 'UNIV2WBTCETH',
+      address: '0x0000000000000000000000000000000000000000'
+    },
+    USDC: {
+      name: 'USDC',
+      address: '0x0000000000000000000000000000000000000000',
+      serviceRegistryName: SERVICE_REGISTRY_NAMES.common.USDC
+    },
+    USDT: {
+      name: 'USDT',
+      address: '0x0000000000000000000000000000000000000000'
+    },
+    WBTC: {
+      name: 'WBTC',
+      address: '0x0000000000000000000000000000000000000000',
+      serviceRegistryName: SERVICE_REGISTRY_NAMES.common.WBTC
+    },
+    WETH: {
+      name: 'WETH',
+      address: '0x0000000000000000000000000000000000000000',
+      serviceRegistryName: SERVICE_REGISTRY_NAMES.common.WETH
+    },
+    WLD: {
+      name: 'WLD',
+      address: '0x0000000000000000000000000000000000000000'
+    },
+    WSTETH: {
+      name: 'WSTETH',
+      address: '0x0000000000000000000000000000000000000000',
+      serviceRegistryName: SERVICE_REGISTRY_NAMES.common.WSTETH
+    },
```

Comment:
Can we override later those addresses in registry?

Reply:
We cannot override in current ServiceRegistry implementation. But this one (or any empty) is not added to SR.